### PR TITLE
#2190 fix note for sending email

### PR DESCRIPTION
**Pull request by neetapimpalnerkar**
Please review #2190.

### CODE REVIEW

File: source_en/ArchitectureInDetail/MessagingDetail/Email.rst [hunk 1]
```diff
@@ -1013,25 +1013,25 @@ garbling occurs for seven characters described in the table below.
...
      - | 2141
-     - | 〜(Tilde)
+     - | ~ (Tilde)
    * - | ∥ (Parallel symbol)
      - | 8161
      - | U+2225
      - | U+2016
      - | 2142
-     - | ‖ (Pipe sumbol)
+     - | || (Pipe sumbol)
```
Comment (btundes) [inline source_en/ArchitectureInDetail/MessagingDetail/Email.rst]: @asaharas / @neetapimpalnerkar 
L1034 - Please correct the spelling of word symbol.

**Comment by neetapimpalnerkar** [comment]
@btundes  Original Japanese file shows garbage character instead of pipe. I have tried to change it again.

**Comment by asaharas** [comment]
refer to #2709

**Comment by asaharas** [comment]
@neetapimpalnerkar 
you didn't need to change it.
because of wrong character encoding, you saw the garbage character.
and I have fixed it.

**Pull request merged**